tests/test_command.py

```
@@ -1430,6 +1469,18 @@ def test__create_cmd_auto_machine_id(self):
         r = boom.command._create_cmd(args, None, opts, None)
         self.assertNotEqual(r, 1)
 
+    @unittest.skipIf(not have_root_lv(), "requires root LV")
```

Ruff (0.12.2)

`have_root_lv` may be undefined, or defined from star imports

(F405)